tests/Fixtures/TestBundle/Entity/ArrayFilterValidator.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Filter\ArrayRequiredFilter;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Filter Validator entity.
+ *
+ * @author Julien Deniau <julien.deniau@gmail.com>
+ *
+ * @ApiResource(attributes={
+ *     "filters"={
+ *         ArrayRequiredFilter::class
+ *     }
+ * })
+ * @ORM\Entity
+ */
+class ArrayFilterValidator
+{
+    /**
+     * @var int The id
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string A name
+     *
+     * @ORM\Column
+     * @ApiProperty(iri="http://schema.org/name")
+     */
+    private $name;
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function setId($id)
+    {
+        $this->id = $id;
+    }
```

Comment:
For simplicity's sake you could have put public properties here, that's find as-is anyway! 
Although, are you sure you want `id` to be writable?

Reply:
Sure thing! Just remove the id setter if it's auto generated though :)